src/lib/features/client-feature-toggles/tests/client-feature-toggles.e2e.test.ts

```diff
@@ -169,7 +173,11 @@ test('should return correct data structure from /api/client/features', async ()
         {
             name: 'flexibleRollout',
             constraints: [],
-            parameters: { rollout: '100' },
+            parameters: {
```

Comment:
Ironically the test name of ''should return correct data structure from /api/client/features" was wrong but I've updated both the call for the data and the data that we check against. For this test to previously be correct, the data it matched against should have included stickiness and groupId

Reply:
Shouldn't the request body remain as before? or maybe empty? Perhaps there's a test for that already

Reply:
Was going to comment that if we don't already have that, we should. Essentially a test that validates that adding a strategy without all the parameters still grabs the default parameters.